### DISCUSSION TITLE
Frigate in Docker Compose keeps restarting

### DISCUSSION THREAD

**Discussion by isaacronquest** [blakeblackshear/frigate#5442] [Ask A Question]
New to the developers world. After a steep learning curve, I managed to get Frigate installed in Docker with Docker Compose. After starting docker compose, Frigate just keeps restarting over and over. See log below

richard@Beelink-MINI-S:~/docker-compose$ sudo docker compose up --remove-orphans
[+] Running 2/0
 ⠿ Container docker-compose-mosquitto-1  Removed                                                                                                         0.0s
 ⠿ Container frigate                     Created                                                                                                         0.0s
Attaching to frigate
frigate  | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
frigate  | [s6-init] ensuring user provided files have correct perms...exited 0.
frigate  | [fix-attrs.d] applying ownership & permissions fixes...
frigate  | [fix-attrs.d] done.
frigate  | [cont-init.d] executing container initialization scripts...
frigate  | [cont-init.d] done.
frigate  | [services.d] starting services
frigate  | [services.d] done.
frigate  | [2023-02-09 23:00:15] frigate.app                    INFO    : Starting Frigate (0.11.1-2eada21)
frigate  | Starting migrations
frigate  | [2023-02-09 23:00:15] peewee_migrate                 INFO    : Starting migrations
frigate  | There is nothing to migrate
frigate  | [2023-02-09 23:00:15] peewee_migrate                 INFO    : There is nothing to migrate
frigate  | [Errno -2] Name or service not known
frigate  | [2023-02-09 23:00:15] frigate.mqtt                   ERROR   : Unable to connect to MQTT server: [Errno -2] Name or service not known
frigate  | [cmd] python3 exited 1
frigate  | [cont-finish.d] executing container finish scripts...
frigate  | [cont-finish.d] done.
frigate  | [s6-finish] waiting for services.
frigate  | [s6-finish] sending all processes the TERM signal.
frigate  | [s6-finish] sending all processes the KILL signal and exiting.
frigate exited with code 1
frigate  | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
frigate  | [s6-init] ensuring user provided files have correct perms...exited 0.
frigate  | [fix-attrs.d] applying ownership & permissions fixes...
frigate  | [fix-attrs.d] done.
frigate  | [cont-init.d] executing container initialization scripts...
frigate  | [cont-init.d] done.
frigate  | [services.d] starting services
frigate  | [services.d] done.
frigate  | [2023-02-09 23:00:21] frigate.app                    INFO    : Starting Frigate (0.11.1-2eada21)
frigate  | Starting migrations
frigate  | [2023-02-09 23:00:21] peewee_migrate                 INFO    : Starting migrations
frigate  | There is nothing to migrate
frigate  | [2023-02-09 23:00:21] peewee_migrate                 INFO    : There is nothing to migrate
frigate  | [Errno -2] Name or service not known
frigate  | [cmd] python3 exited 1
frigate  | [cont-finish.d] executing container finish scripts...
frigate  | [cont-finish.d] done.
frigate  | [s6-finish] waiting for services.
frigate  | [s6-finish] sending all processes the TERM signal.
frigate  | [s6-finish] sending all processes the KILL signal and exiting.
frigate exited with code 1

Any advice?
<img width="1433" alt="Screenshot 2023-02-09 at 23 10 23" src="https://user-images.githubusercontent.com/110307173/217940152-d795ab09-a950-4a34-a397-006f321f2ea8.png">

**Comment by NickM-27** [comment]
it can't find your mqtt server, what is your config?
